tensorflow/compiler/tf2xla/const_analysis.cc

```diff
@@ -74,8 +74,8 @@ Status CondConstInputIndices(
         *(fbody->graph), &compile_time_const_arg_indices,
         /*compile_time_const_nodes=*/nullptr, flib_runtime));
   }
-  for (int i = 0, iter_limit = compile_time_const_arg_indices.size();
-       i < iter_limit; i++) {
+  for (int i = 0, end = compile_time_const_arg_indices.size();
+       i < end; i++) {
```

Comment:
Should merge in a single line

Reply:
handled in previous PR.

Reply:
Keeping open until fully handled, the previous PR lands